assets/i18n/qqq.json

```diff
@@ -575,7 +576,23 @@
 	"I18N_UNSUBSCRIBE_BUTTON_TEXT": "The text that appears on the unsubscribe button, which allows users to unsubscribe from creators.\n{{Identical|Unsubscribe}}",
+    "I18N_WELCOME_GUIDE_TITLE": "Teachers and Parents: A Guide to Oppia",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_1": "Search bar used to search for a specific exploration",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_2": "Category Filter to filter explorations by topic (music, mathematics, biology, etc)",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_3": "Language Filter to filter explorations by language",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_4": "Navigation button that you can click on to browse through more explorations",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_5": "Language Selector used to change the language of the entire page",
+    "I18N_WELCOME_PAGE_COLLECTIONS_PARAGRAPH": "A Collection in Oppia is a library of lessons.  Oppia has several collections, including the “Fractions” collection. Within the “Fractions” collection, there are different explorations that help a student strengthen their fraction skills. ",
+    "I18N_WELCOME_PAGE_COLLECTIONS_TITLE": "What is a Collection?",
+    "I18N_WELCOME_PAGE_EXPLORATION_PARAGRAPH": "At Oppia, the lessons we provide for students are called explorations. All Explorations are comprised of images, videos, and questions that help students explore and strengthen their knowledge on various topics in a fun and interactive way. The first exploration in the Fraction collection is called “What is a Fraction?” ",
+    "I18N_WELCOME_PAGE_EXPLORATION_TITLE": "What is an Exploration?",
+    "I18N_WELCOME_GUIDE_HOW_TO_TITLE": "How Can I Find Explorations and Collections?",
+    "I18N_WELCOME_GUIDE_PARAGRAH_1": "Oppia is an online learning platform that caters specifically to the needs of the world’s underserved youth. As a parent or teacher, we know your involvement is crucial to a child’s education. Here is how you can use Oppia to help your students reach their full potential.",
+    "I18N_WELCOME_GUIDE_PARAGRAH_2":"You can find different explorations and collections on the Exploration Library page. You can also find the Exploration Library by clicking the “Library” button at the top of this page.",
+    "I18N_WELCOME_GUIDE_PARAGRAH_3":"You can filter explorations by different categories including Art, Mathematics,  English and more! You can also look for specific explorations using the search bar. Below each collection there is a rating and also the number of students who have completed that exploration. To help you evaluate its usefulness, each collection has a rating and shows you how many students have used it.",
+    "I18N_WELCOME_GUIDE_PARAGRAH_4":"Below is a snapshot of the library. The highlighted parts are different features of the page you can use to find explorations. Read the descriptions below to understand what these features do.",
```

Comment:
Here should be not translations but an explanation where is thi text located and what it means; look at the other entries in this file.

Reply:
Done!